src/main/java/graphql/nadel/NadelExecutionInput.java

```diff
@@ -16,10 +17,10 @@
     private final Map<String, Object> variables;
 
     private NadelExecutionInput(String query, String operationName, Object context, Map<String, Object> variables) {
-        this.query = query;
+        this.query = requireNonNull(query);
```

Comment:
we must decide if we wanna got with the graphq-java assertions or with java ones.

I used the graphql-java ones so far

Reply:
I saw the Java ones and copied that.  

For null asserts I would use Java ones (post 1.8) but Java 8 does not have the other types so I think a mix of Java NPE ones plus other ones is ok